src/libpriv/rpmostree-unpacker.c

```diff
@@ -750,13 +750,6 @@ import_rpm_to_repo (RpmOstreeUnpacker *self,
       if (!ostree_repo_write_dfd_to_mtree (repo, tmpdir.fd, ".", mtree, modifier,
                                            cancellable, error))
         return FALSE;
-
-      /* check if any of the cbs set an error */
```

Comment:
It can also be set during `write_dfd_to_mtree` through the `modifier`. The callback gets called for each path. This lead to a subtle bug before: https://github.com/projectatomic/rpm-ostree/pull/624.

Alternatively, we can nullify the filter before calling `write_dfd_to_mtree` since we don't actually need filtering functionality. This is probably the better approach; as a plus, we no longer have to ignore this in Coverity (it gets confused because it thinks `cb_error` can never be `NULL` here because of the first check above).